examples/language-sdk-instrumentation/nodejs/express-ts/package.json

```diff
@@ -11,10 +11,10 @@
   "license": "Apache-2.0",
   "dependencies": {
     "@pyroscope/nodejs": "^0.2.x",
-    "axios": "^0.26.1",
+    "axios": "^0.28.0",
     "express": "^4.17.3",
     "morgan": "^1.10.0",
-    "pprof": "^3.2.0",
+    "pprof": "^4.0.0",
```

Comment:
Why do we have to upgrade? Why to a _major_ version?

Reply:
The lock file was unused in the dockerfile and so I resetted the whole lock file.